docusaurus/docs/plugins/plugins-getting-started.md

```diff
@@ -42,30 +36,26 @@ yarn install
 yarn dev
 ```
 
-You should be able to open a browser at `https://127.0.0.1:8005` and you'll get the Rancher Dashboard UI.
-
-This illustrates being able to share the `shell` across UIs - in practice, we would extract the Rancher code from the shell, so instead of getting the Rancher
-functionality in this example, you'd get an empty shell application with the side menu, UI chrome, avatar menu etc.
-
-You would now add new functionality to this application by adding new UI Packages.
+> Note: You will need to have a Rancher backend available and the `API` environment variable set up to reference it.
```

Comment:
I don't know how much hand holding we want to do here, but for ease we could tell them to run API=xyz yarn dev?

Reply:
Made a small tweak